client/me/memberships/subscription.jsx

```diff
@@ -0,0 +1,138 @@
+import PurchasesHeader from '../purchases/purchases-list/header';
+import Main from 'components/main';
+import DocumentHead from 'components/data/document-head';
+import QueryMembershipsSubscriptions from 'components/data/query-memberships-subscriptions';
+import formatCurrency from 'lib/format-currency';
+import HeaderCake from 'components/header-cake';
+import { purchasesRoot } from '../purchases/paths';
+import Site from 'blocks/site';
+import Gridicon from 'gridicons';
+import CompactCard from 'components/card/compact';
+import { requestSubscriptionStop } from 'state/memberships/subscriptions/actions';
+import Notice from 'components/notice';
+
+class Subscription extends React.Component {
+	constructor() {
+		super();
+		this.stopSubscription = () => this.props.requestSubscriptionStop( this.props.subscription.ID );
+	}
+	render() {
+		const { translate, subscription, moment, stoppingStatus } = this.props;
+		return (
+			<Main className="memberships__subscription">
+				<DocumentHead title={ translate( 'My Memberships' ) } />
+				<MeSidebarNavigation />
+				<QueryMembershipsSubscriptions />
+				<PurchasesHeader section={ 'memberships' } />
+				<HeaderCake backHref={ purchasesRoot + '/memberships' }>
+					{ subscription ? subscription.title : translate( 'All subscriptions' ) }
+				</HeaderCake>
+				{ stoppingStatus === 'start' && (
+					<Notice
+						status="is-info"
+						isLoading={ true }
+						text={ translate( 'Stopping this subscription' ) }
+					/>
+				) }
+				{ stoppingStatus === 'fail' && (
+					<Notice
+						status="is-error"
+						text={ translate( 'There was a problem while stopping your subscription' ) }
+					/>
+				) }
+				{ stoppingStatus === 'success' && (
+					<Notice
+						status="is-success"
+						text={ translate( 'This subscription has been stopped. You will not be charged.' ) }
+					/>
+				) }
+
+				{ subscription && (
+					<div>
+						<Card className="memberships__subscription-meta">
+							<Site siteId={ subscription.site_id } href={ subscription.site_url } />
+							<div className="memberships__subscription-title">{ subscription.title }</div>
+							<Fragment>
+								<ul className="memberships__subscription-inner-meta">
+									<li>
+										<em className="memberships__subscription-inner-detail-label">
+											{ translate( 'Price' ) }
+										</em>
+										<span className="memberships__subscription-inner-detail">
+											{ formatCurrency( subscription.renewal_price, subscription.currency ) }
+										</span>
+									</li>
+									<li>
+										<em className="memberships__subscription-inner-detail-label">
+											{ translate( 'Renew interval' ) }
+										</em>
+										<span className="memberships__subscription-inner-detail">
+											{ subscription.renew_interval }
+										</span>
+									</li>
+									<li>
+										<em className="memberships__subscription-inner-detail-label">
+											{ translate( 'Subscribed on' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **37** times:
`translate( 'Subscribed On' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).